src/app/components/SocialMedia/index.tsx

```diff
@@ -1,16 +1,13 @@
 import styled from "styled-components/macro";
-import { ReactComponent as InstaIcon} from '../../../../assets/images/Instagram.svg';
-import { ReactComponent as FacebookIcon } from '../../../../assets/images/Facebook.svg';
-import { ReactComponent as YoutubeIcon} from '../../../../assets/images/YouTube.svg';
+import { ReactComponent as InstaIcon} from '../../../assets/images/Instagram.svg';
+import { ReactComponent as FacebookIcon } from '../../../assets/images/Facebook.svg';
+import { ReactComponent as YoutubeIcon} from '../../../assets/images/YouTube.svg';
+
 // import {FunctionComponent} from 'react';
 
 export const MediaContainer = styled.div`
```

Comment:
You are inconsistent with wrappers. I suggest you call them `...Styled`. There is a `MediaContainer` and there is `MediaContainerStyled`. Make sure you renamed it everywhere